Mathlib/LinearAlgebra/StdBasis.lean

```diff
@@ -105,7 +82,7 @@ given by `s j` on each component.
 For the standard basis over `R` on the finite-dimensional space `η → R` see `Pi.basisFun`.
 -/
 protected noncomputable def basis (s : ∀ j, Basis (ιs j) R (Ms j)) :
-    Basis (Σj, ιs j) R (∀ j, Ms j) :=
+    Basis (Σ j, ιs j) R (∀ j, Ms j) :=
```

Comment:
By the way: this change should disappear when merging master (as it was already made on master).

Reply:
It's still here ... What PR is that?